x-pack/plugins/security_solution/public/common/containers/use_full_screen/index.tsx

```diff
@@ -47,10 +47,10 @@ export const useGlobalFullScreen = (): GlobalFullScreen => {
       const isDataGridFullScreen = document.querySelector('.euiDataGrid--fullScreen') !== null;
       if (fullScreen) {
         document.body.classList.add(SCROLLING_DISABLED_CLASS_NAME, 'euiDataGrid__restrictBody');
-        resetScroll();
+        // resetScroll();
```

Comment:
@andrew-goldstein do you remember why this was added originally? imo the app feels better without it

Reply:
looks like it was added to fix this bug https://github.com/elastic/kibana/issues/73882 which is no longer relevant